model/task_lifecycle.go

```diff
@@ -725,14 +728,10 @@ func tryDequeueAndAbortCommitQueueVersion(t *task.Task, cq commitqueue.CommitQue
 	}
 
 	issue := p.Id.Hex()
-	if p.IsPRMergePatch() {
```

Comment:
this possibly fixes a weird error I found when sometimes we have trouble finding pr patches in the commit queue. I have no explanation for why it happened but we don't need to do this anyway